README.md

```diff
@@ -911,6 +911,9 @@ arXiv, 2023. [PDF](https://arxiv.org/pdf/2305.06382), [Project Page](https://erc
 - <a name="Zhang23arxiv2"></a>Zhang, P., Liu, H., Ge, Z., Wang, C., Lam, E. Y.,   
 *[Neuromorphic Imaging with Joint Image Deblurring and Event Denoising](https://arxiv.org/abs/2309.16106)*,  
 arXiv, 2023.
+- <a name="liu2023tpami"></a>Liu, S., Dragotti, P.L., 
```

Comment:
![image](https://github.com/uzh-rpg/event-based_vision_resources/assets/8024432/567afc02-3289-4059-a9e1-4928065c4c44)

Please add line breaks: Authors, Title and Publication venue in 3 separate lines. Thanks.